### DISCUSSION TITLE
when are upgrading new version to fix vulnerability

### DISCUSSION THREAD

**Discussion by bhavink25** [xunit/xunit#2766] [Feedback Request]
current packages need to upgrade System.Net.Http@4.3.0

 xunit@2.5.0 > xunit.core@2.5.0 > xunit.extensibility.execution@2.5.0 > xunit.extensibility.core@2.5.0 > NETStandard.Library@1.6.1 > System.Net.Http@4.3.0

**Comment by bradwilson** [comment]
Never.

We do not depend on this library. This is a transient dependency (from .NET Standard) that we don't control, and none of xUnit.net ever consumes `System.Net.Http`.

You are free to manually add a dependency for a later version if your code does depend on this.

(Also, please never use xUnit.net in production systems.)
